code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -292,7 +292,7 @@
 	var/area/joined_area = get_area(new_character.loc)
 	if(joined_area)
 		joined_area.on_joining_game(new_character)
-	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_CREWMEMBER_JOINED, new_character, new_character.mind.assigned_role.title)
+//	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_CREWMEMBER_JOINED, new_character, new_character.mind.assigned_role.title) //monkestation edit, this makes this signal send twice
```

Comment:
are you certain this has no repercussions?

Reply:
No, but I dont see a reason for this signal being added in the PR that did so and its definitely a duplicate.

Reply:
had to remove the other one